src/current/docs-mcp-integration.md

```diff
@@ -0,0 +1,193 @@
+---
+title: CockroachDB Docs MCP
+summary: Connect to CockroachDB documentation directly from your IDE or AI assistant
+toc: true
+docs_area: reference
+---
+
+The CockroachDB Docs MCP (Model Context Protocol) server enables AI assistants like Claude Desktop, Cursor, and VS Code to access CockroachDB documentation directly. This allows you to get instant, context-aware answers about CockroachDB without leaving your development environment.
+
+## Create your own MCP
+
+Want to build your own MCP server? Check out the [MCP documentation](https://modelcontextprotocol.io/introduction) to get started.
+
+## Setup
+
+Connect your AI assistant to CockroachDB documentation by configuring the MCP server. Choose your platform below for specific instructions.
+
+### Cursor
+
+1. Open Cursor Settings (⌘+,)
+2. Navigate to **Models** → **Model Context Protocol**
+3. Add the following configuration:
+
+```json
```

Comment:
same comment as elsewhere re: the code block needs indentation for the numbered list to render properly (try 4 spaces)